dataprofiler/tests/profilers/test_column_profile_compilers.py

```diff
@@ -500,12 +500,13 @@ def test_column_stats_profile_compiler_stats_diff(self):
                 "categories": [["1"], ["9"], ["10"]],
                 "gini_impurity": 0.06944444444444448,
                 "unalikeability": 0.16666666666666663,
-                "categorical_count": {"9": -1, "1": [1, None], "10": [None, 1]},
+                "categorical_count": {"9": -1, "1": 1, "10": -1},
```

Comment:
@ksneab7 @micdavis I'm uncomfortable with this changing in the expected_diff but the more I look at it, it appears to be correct, but again changing expected values like this in the diff is uncomfortable... so if you do research something more in-depth take an extra look at this

Reply:
actually I found the justification for this change and it lies in `profiler_utils.find_diff_of_dicts` ... on L577, we do not hit that any more due to both `cat_count` dictionaries having all the categories between both profiles